### PR TITLE
fix(PPDSC-2611): fix pipeline

### PR DESCRIPTION
**Pull request by mstuartf**
[PPDSC-2611](https://nidigitalsolutions.jira.com/browse/PPDSC-2611)

**What**

1. Background - why this is needed
- Error in main job script
2. What did you do
- Prevent logged values being returned by check updates script
3. What does the reviewers should expect
- Baselines should update correctly

Updates required case:
Screenshot 2023-01-31 at 11 42 30

No updates required case:
Screenshot 2023-01-31 at 11 50 33

Branch not found case:
Screenshot 2023-01-31 at 15 58 24

Token not found case:
Screenshot 2023-01-31 at 15 58 10

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

[PPDSC-2611]: https://nidigitalsolutions.jira.com/browse/PPDSC-2611?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2611-fix
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2611-fix/storybook

File: scripts/check-baseline-updates.js [hunk 1]
```diff
@@ -48,7 +55,7 @@ async function getPercyBuildForBranch(branchName) {
   throw Error();
```
Comment (Xin00163) [inline scripts/check-baseline-updates.js:55]: Do you think we could add the error message in throw? In the last test, this log is not printed.

Comment (JohnTParsons) [reply scripts/check-baseline-updates.js:55]: Good suggestion

Comment (mstuartf) [reply scripts/check-baseline-updates.js]: Updated. It will now always log the error and the typo in the loop (`<` instead of `<=` is fixed).

**Review by JohnTParsons — APPROVED** [review]

**Review by Xin00163 — APPROVED** [review]

**Pull request merged**